Azure/azure-cli
# [AppService] Fix az webapp create - use hardcoded runtime stacks

**Description**  
Fix `az webapp create` to use the updated hardcoded runtimes. 
Closes #14281 

A while back as part of issue #13261, `az webapp list-runtimes` was updated to a hardcoded list. `az webapp create` implementation wasn't updated to use this hardcoded list, so this PR fixes it

**Testing Guide**  
`az webapp create -n <webappname> -g <resource_group_name> -l -p <app_service_plan> -r "DOTNETCORE|3.1"` should work, since DOTNETCORE|3.1 is a new runtime added in the hardcoded list-runtimes command

`az webapp create -n <webappname> -g <resource_group_name> -l -p <app_service_plan> -r "node|6.6"` should not work, since node|6.6 is a deprecated runtime

---

This checklist is used to make sure that common guidelines for a pull request are followed.
